python/cudf/cudf/core/buffer.py

```diff
@@ -33,21 +33,20 @@ class Buffer(Serializable):
         object is kept in this Buffer.
     """
 
-    ptr: int
     size: int
```

Comment:
Could we make `size` a private attribute `_size`? It seems like it has a public name `nbytes` already. This class shouldn't expose a mutable public attribute `size` as well as an immutable public property `nbytes`.